src/main/resources/api.yaml

```diff
@@ -0,0 +1,174 @@
+swagger: '2.0'
+info:
+  description: >-
+    This documents the OTP Middleware endpoints. For this sample, you can use
+    the api key `special-key` to test the authorization filters.
+  version: 0.0.0
+  title: OTP Middleware
+  termsOfService: 'http://fdot-otp-support.ibi-transit.com/terms'
+  contact:
+    url: 'https://github.com/ibi-group/otp-middleware/issues'
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+host: fdot-otp-support.ibi-transit.com
+basePath: /
+tags:
+  - name: user
+    description: Operations about User
+schemes:
+  - https
+  - http
+paths:
+  '/user/{id}':
+    get:
+      tags:
+        - user
+      summary: Get OTP user by ID
+      description: ''
+      operationId: getOne
+      produces:
+        - application/json
+      parameters:
+        - name: id
+          in: path
+          description: The ID that needs to be fetched.
+          required: true
+          type: string
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/User'
+        '400':
+          description: Invalid username supplied
+        '404':
+          description: User not found
+    put:
+      tags:
+        - user
+      summary: Updates user info
+      description: This can only be done by the logged in user.
```

Comment:
A lot of these values seem more like placeholders than anything. I think we should go through and either remove much of them or add TODO comments.

Reply:
Yes, that is just a starter, copy of the example in swagger-ui to set up the documentation processes. Feel free to make corrections.